string robot_name
int32 x_pos
int32 y_pos
int32 z_pos